File: javascript/fizzbuzz.js
'''
Fizzbuzz

Input: list of integers
Output: Write a script that will print 1-100 except:

if divisable by 3 - write fizz
if divisable by 5 - write buzz
if divisable by 15 - write fizz buzz
'''

var list_of_numbers = [1, 2, 3, 4, 5, 6, 15, 28, 30, 33, 35, 36, 40, 42];

function fizzbuzz(list_of_numbers) {
	for (var i=0; i<list_of_numbers.length; i++) {
		var j = list_of_numbers[i]
		if (j%3==0 && j%5==0) {
			console.log(j, "fizz buzz");
		}
		else if (i%3==0) {
			console.log(j, "fizz");
		}
		else if (i%5==0) {
			console.log(j, "buzz");
		}
		else {
			console.log(j);
		}
	}
}

// Test section
fizzbuzz(list_of_numbers);

File: java/javacode/FizzBuzz.java
// Fizz Buzz

// Input: list of numbers

// Output: print numbers submitted except:
//   if divisable by 3 - write fizz
//   if divisable by 5 - write buzz
//   if divisable by 15 - write fizz buzz


import java.util.Scanner;

public class FizzBuzz
{
    public static void main( String[] args)
    {
        Scanner keyboard = new Scanner(System.in);

        int num;

        boolean fizzed;

        System.out.print("Enter a number for printing fizzbuzz:");
        num = keyboard.nextInt();

        for (int i = 1; i <= num; i++) {
            // System.out.println(i);

            if (i % 3 == 0 && i % 5 == 0)
                System.out.println("fizz buzz");
            else if (i % 3 == 0 )
                System.out.println("fizz");
            else if (i % 5 == 0)
                System.out.println("buzz");
            else
                System.out.println(i);
        }
    }
}